packages/terra-profile-image/src/ProfileImage.jsx

```diff
@@ -16,6 +16,11 @@ const propTypes = {
    * The text content that specifies an alternative text for an image.
    */
   alt: PropTypes.string,
+  /**
+  * Sets the `object-fit` style of the image from the following values: `cover`, `contain`, `scale-down`, `none`.
+  * ![IMPORTANT](https://badgen.net/badge/UX/Design-Standards/blue) Anywhere the terra-profile-image is used to show images of People, _only_ `cover` and `contain` are acceptable.
+  */
+  fit: PropTypes.oneOf(['cover', 'scale-down', 'contain', 'none']),
```

Comment:
Why is "fill" not an option for ProfileImage?

Reply:
It was decided by engineering to remove the `fill` prop option based on UX feedback that: 

>  for Profile Images that are intended to be photos of People - only `cover` and `contain` would be the acceptable values. `fill` would never be acceptable by UX as is changes the original image aspect ratio and distorts the face and likeness of the image.

Comment on the original issue here: [Issue #2798(comment)-584576876](https://github.com/cerner/terra-core/issues/2798#issuecomment-584576876) and PR here: [PR#2804(comment)-356539191](https://github.com/cerner/terra-core/pull/2804#pullrequestreview-356539191)

`Cover` is the recommended UX styling prop to use for profile-images and will match how it is currently built in new Avatar, the replacement for profile-image (consumers do not have the option to adjust it in avatar).